jpf.properties

```diff
@@ -1,40 +1,10 @@
-#--- JPF project properties file for project jpf-nas
-jpf-nas = ${config_path}
-
-#--- other JPF projects we depend on
-@using jpf-core
-
-
-#--- project specific host VM classpath (used by Java to load classes)
-# NOTE: you have to replace the wildcard jar specs with explicit jar pathnames
-# if this property file is used from within a build.xml or NetBeans project.xml,
-# since both do not support wildcard patterns in classpath specs
 jpf-nas.native_classpath =\
-   ${jpf-nas}/build/jpf-nas.jar;\
+   ${jpf-nas}/build/libs/jpf-nas.jar;\
```

Comment:
The old build produced two JAR file with classes; one at the JPF-native level (JPF's "Java Model Interface"), and one at the library level (model classes).
Please try to fix the build, and once you have the path names resolved, I can try to run the new version on an example to see if this setup works.